src/Adapter/Driver/Pdo/Connection.php

```diff
@@ -237,6 +240,9 @@ public function connect()
                     if (isset($charset) && $pdoDriver != 'pgsql') {
                         $dsn[] = "charset={$charset}";
                     }
+                    if (isset($unix_socket)) {
```

Comment:
Actually, it might be better to throw an exception in that case; rather than leave it to undefined behavior.